_data/backup.yml

```
@@ -135,7 +135,7 @@ websites:
     email: Yes
     sms: Yes
     software: Yes
-    doc: http://support.nimbox.co.uk/knowledge_base/topics/two-step-slash-two-factor-authentication
+    doc: https://kb-80910.groovehq.com/help/two-step-slash-two-factor-authentication
```

```suggestion
    doc: https://www.nimbox.co.uk/help-centre/two-step-two-factor-authentication-2fa/
```
I'd go for the Help Centre on the page directly, rather than the forward to groovehq.